### PR TITLE
decapitalizing Authorization string

### PR DESCRIPTION
**Pull request by abuZayed15**
This PR is for issue #189 

I have decapitalized the authorization token `$"Ubi_v1 t` to `$"ubi_v1 t=`

Thank you for allowing me to contribute to this issue.

### CODE REVIEW

**Comment by abuZayed15** [comment]
Sorry, I must have missed out on one more authorization string. Let me check and push again

File: DragonFruit.Six.API/Dragon6Client.cs [hunk 1]
```diff
@@ -51,7 +51,7 @@ protected Dragon6Client()
             }
         }
 
-        #endregion
+        #endregion Constructors
```
Comment (aspriddell) [inline DragonFruit.Six.API/Dragon6Client.cs:54]: This line will cause a compiler fail

**Review by aspriddell — CHANGES_REQUESTED** [review]
See review

**Comment by aspriddell** [comment]
Superseded by #192

**Pull request closed without merging**